recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+c_compiler_version:    # [linux and not aarch64]
+- '8'                  # [linux and not aarch64]
+cxx_compiler_version:  # [linux and not aarch64]
+- '8'                  # [linux and not aarch64]
```

Comment:
C++17 was still marked as experimental in gcc 8, IINM. So there may be incompatible changes in libstdc++ after that.

Reply:
You are correct, C++17 remained experimental in gcc8. I have updated the compiler to gcc9 where available as well as documenting the reason for the compiler call out.